src/file_tree.ml

```diff
@@ -106,7 +106,6 @@ end
 
 type t =
   { root : Dir.t
```

Comment:
In a degenerate case like this, I think that it's more common to use a variant to encode the newtype. What do you think about `type t = File_tree of Dir.t`?

Reply:
I was planning on just making it a type alias actually. Thanks for reminding